kleros-sdk/test/dataMappings.test.ts

```diff
@@ -33,7 +33,7 @@ describe("jsonAction", () => {
 
 describe("subgraphAction with variables", () => {
   it("should fetch GraphQL data with variables and return in expected format", async () => {
-    const endpoint = "https://api.thegraph.com/subgraphs/name/kemuru/escrow-v2-devnet";
+    const endpoint = "https://api.studio.thegraph.com/query/61738/escrow-v2-devnet/version/latest";
```

Comment:
Let's make the user provide their own Graph API key so people don't use the Studio endpoint